photon-server/src/main/resources/web/index.html

```diff
@@ -1 +1 @@
-<p>UI has not been copied!</p>
+<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><link rel="icon" href="/favicon.png"><title>PhotonVision</title><link href="/js/chunk-2d216214.90e8fa62.js" rel="prefetch"><link href="/js/chunk-2d216257.8e8f2e7b.js" rel="prefetch"><link href="/js/chunk-fdf80f12.244b6d05.js" rel="prefetch"><link href="/css/app.53f364c5.css" rel="preload" as="style"><link href="/css/chunk-vendors.a79457dc.css" rel="preload" as="style"><link href="/js/app.8e11f439.js" rel="preload" as="script"><link href="/js/chunk-vendors.7c9bd682.js" rel="preload" as="script"><link href="/css/chunk-vendors.a79457dc.css" rel="stylesheet"><link href="/css/app.53f364c5.css" rel="stylesheet"></head><body><noscript><strong>We're sorry but PhotonVision doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id="app"></div><script src="/js/chunk-vendors.7c9bd682.js"></script><script src="/js/app.8e11f439.js"></script></body></html>
```

Comment:
ope

Reply:
...

Reply:
How do I stop this from updating every time? Just adding it to gitignore?